### ISSUE TITLE
evaluate finetuned models

### ISSUE THREAD

**Issue by Wafaa014** [rrmenon10/ADAPET#22]
I am trying to evaluate your fine-tuned model on BoolQ task. I am using this command :

`python src/run_pretrained.py -m {finetuned_model_dir}/{task_name} -c config/{task_name}.json -k pattern={best_pattern_for_task}
`

when trying to run it I get the following warning:

`Token indices sequence length is longer than the specified maximum sequence length for this model (737 > 512). Running this sequence through the model will result in indexing errors`

I want to know why is this happening? and how can I fix it to get the evaluation results?

**Note:** I tried with another task (CB), and it is working fine.

**Comment by rrmenon10** [comment]
Hi @Wafaa014, I tried to run this code from scratch on my end and couldn't replicate this issue. Could you provide the exact command you had used for this? I tried the command on my end and found no problems. I also went through the BoolQ validation examples and found that all examples were well within the 512 sequence length when applying the entire pattern. You may want to have a look at the examples being used in this case. Hope this helps!

**Comment by Wafaa014** [comment]
Hello @rrmenon10, thank you for the response. I have used the same command you provided: 

`python src/run_pretrained.py -m {finetuned_model_dir}/{task_name} -c config/{task_name}.json -k pattern={best_pattern_for_task}`

and I used the same BoolQ dataset in the SuperGlue and FewGlue folders.

**Comment by rrmenon10** [comment]
Hi @Wafaa014, could you perhaps also provide the examples in the BoolQ dataset for which you encounter this issue? If you add the line `if pet_mask_ids.shape[-1] > 512: print(batch["input"]["idx"])` after [this line](https://github.com/rrmenon10/ADAPET/blob/b36739cdd95bc8412ec017cfb47d753946ef0b77/src/adapet.py#L350), you should be able to see the indices of the validation set that result in the error.

**Comment by rrmenon10** [comment]
Closing due to inactivity.
